scripts/convert_csv_to_json.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import os
+import pandas as pd
+import sys
+import json
+
+
+def generate_json_files(version):
```

Comment:
It could do, but the business don't care about formal versions of the files so it would be us creating a version scheme anyway. They won't be including version info within the file and asking them to put it in the filename is fraught with error. I did consider auto creating a version folder but didn't think it was worth it for this. This is more along the lines of 'which sub folder do you want to send the output to?'. The current approach is simply saying let's use versions as the subfolder name as it gives us some option when publishing to the CDN